apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx

```diff
@@ -110,6 +111,23 @@ export const TemplatesStoreModal = ({ general, popular, isOpened, onClose }: ITe
     >
       <ModalBodyHolder data-test-id="templates-store-modal">
         <TemplatesSidebarHolder data-test-id="templates-store-modal-sidebar">
+          <TemplatesGroup key="blank-workflow">
+            <GroupName>Blank Workflow</GroupName>
+            <TemplateItem
+              key="temp-blank-workflow"
+              onClick={() => {
+                segment.track('[Template Store] Click Create Notification Template', {
+                  templateIdentifier: 'Blank Workflow',
+                  location: TemplateCreationSourceEnum.DROPDOWN,
+                });
+                handleRedirectToCreateBlankTemplate(false);
+              }}
+            >
+              <FontAwesomeIcon icon={faFile} />
+              <span>Blank Workflow</span>
+            </TemplateItem>
+          </TemplatesGroup>
+          <EchoProjectModalItem />
```

Comment:
I can suggest renaming the `Blank Workflow` group to the `Workflow` and adding the new Echo item under it, looks much better IMO.
![Screenshot 2024-03-12 at 10 17 15](https://github.com/novuhq/novu/assets/2607232/c6e5268e-c24a-4b50-86cd-b55344ac3510)